styles/main.css

```diff
@@ -0,0 +1,230 @@
+}
+
+.nav {
+  text-decoration: none;
+  font-size: 1.4rem;
+  font-weight: 500;
+  color: var(--black);
+}
+
+.nav:hover {
+  color: var(--primary-color);
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  gap: 2rem;
+}
+
+.section {
+  /* margin: 6rem; */
+  margin-top: 6rem;
+  color: var(--black);
+}
+
+.section-main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  border: #ef4444;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  font-size: 3rem;
+}
+
+@media screen and (max-width: 768px) {
+  .container {
+    font-size: 2rem;
+  }
+}
+
+@media screen and (max-width: 640px) {
+  .container {
+    font-size: 1.6rem;
+  }
+}
+
+@media screen and (max-width: 480px) {
+  .container {
+    font-size: 1.2rem;
+  }
+}
+
+.about {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  font-size: 1.5rem;
+  font-weight: 500;
+  margin: 3rem 5rem 3rem 5rem;
+}
+
+.btn-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 5rem;
+}
+
+.btn {
+  text-decoration: none;
+  background: transparent;
+  font-size: 1.2rem;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 1rem 2rem;
+  border-radius: 10px;
+}
+
+.btn-primary {
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn-primary:hover {
+  background-color: var(--primary-color);
+  color: var(--lynx-white);
+}
+
+.btn-secondary {
+  background-color: var(--primary-color);
+  color: var(--lynx-white);
+}
+
+.btn-secondary:hover {
+  background-color: #ef4444;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--white);
+  color: var(--black);
+  box-shadow: 0px -4px 10px rgb(0 0 0 / 5%);
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  min-height: 5rem;
+}
+
+.author {
+  color: var(--primary-color);
+  font-size: 1.2rem;
+}
+
+.socials {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  margin-top: 1rem;
+}
+
+.bi {
+  padding: 1rem;
+  color: var(--black);
+}
+
+.bi:hover {
+  color: var(--primary-color);
+}
+
+/* Documentation  CSS*/
+
+.main-container {
+  background-color: var(--lynx-white);
+  width: 15rem;
+}
+
+.subtext {
+  padding: 1rem;
+  font-size: 1.5rem;
+  font-weight: 500;
+  border-bottom: 1px solid var(--black);
+}
+
+.items {
+  width: 15rem;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  font-size: 1.2rem;
+  overflow: scroll;
+  overflow-y: auto;
+  height: 90vh;
+  overflow-x: hidden;
+  position: absolute;
+}
+
+.item {
+  padding: 1rem;
+  text-decoration: none;
+  color: var(--black);
+}
+
+.item:hover {
+  color: var(--primary-color);
+  background-color: var(--lynx-gray);
+}
+
```

Comment:
lynx grey?

Reply:
It was name for dark shade color for grey. I have changed that.